umkm/inc/TailLight.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+/** Tail-light feature report as observed in USBPcap/Wireshark. */
+
+#include "kmcodesegs.h"
+
+#define TAILLIGHT_SET_COLOR_REPORT_ID 36
+#define TAILLIGHT_UNKNOWN_1 0xB2
+#define TAILLIGHT_UNKNOWN_2 0X03
+
+// Seems compiler is producing things correctly without changing the packing
+struct TailLightReport {
```

Comment:
Is there any specific technical reason for reimplementing the `TailLightReport` struct instead of using the existing one?

If yes, then please submit a separate pull-request where you propose to fix the concrete technical problem.
If not, then please use the existing `TailLightReport` struct instead of reimplementing it.